services/Reddit.json

```
@@ -3,11 +3,17 @@
     "documents": {
         "tos": {
             "location": "https://www.redditinc.com/policies/user-agreement",
-            "contentSelector": "#content"
+            "contentSelector": "#content",
+            "filters": [
+                "removeOptionsList"
+            ]
         },
         "privacy": {
             "location": "https://www.redditinc.com/policies/privacy-policy",
-            "contentSelector": "div#content"
+            "contentSelector": "div#content",
```

When you target with an ID, the tag is (in the vast majority of cases, and theoretically always) not needed, since an ID is expected to be unique in the page.

```suggestion
            "contentSelector": "#content",
```